dumper/lib-dumper-spi/build.gradle

```diff
@@ -19,6 +19,6 @@ plugins {
 }
 
 dependencies {
-	api "com.fasterxml.jackson.core:jackson-databind"
+	implementation "com.fasterxml.jackson.core:jackson-databind"
```

Comment:
Gradle wouldn't pick the pinned versions in `buildSrc/.../conventions` for `api`